spec/features/audit/report/allocation_spec.rb

```diff
@@ -1,34 +1,38 @@
 RSpec.feature "Content Allocation", type: :feature do
   scenario "Filter allocated content" do
-    given_i_am_auditing_a_content_item
-    then_i_see_all_content_items
-    given_that_i_filter_by_attributes_that_do_not_apply_to_any_content_items
-    then_i_see_no_content_items
+    given_i_am_an_auditor_belonging_to_an_organisation
+    when_i_visit_the_report_page
+    then_i_can_see_the_content_item_i_have_been_allocated
+    when_i_select_no_one_in_the_allocated_to_filter
+    then_i_cannot_see_the_content_item_i_have_been_allocated
   end
 
-  def given_i_am_auditing_a_content_item
+  def given_i_am_an_auditor_belonging_to_an_organisation
     organisation = create(:organisation)
     user = create(:user, organisation: organisation)
-    content_item = create(
+    @content_item = create(
       :content_item,
       title: "Do Androids Dream of Electric Sheep",
       allocated_to: user,
       primary_publishing_organisation: organisation,
     )
+  end
+
+  def when_i_visit_the_report_page
     @audit_report_page = ContentAuditTool.new.audit_report_page
-    @audit_report_page.load(content_id: content_item.content_id)
+    @audit_report_page.load(content_id: @content_item.content_id)
   end
 
-  def then_i_see_all_content_items
+  def then_i_can_see_the_content_item_i_have_been_allocated
     expect(@audit_report_page).to have_report_section(text: '1')
```

Comment:
Why are we expecting this text? It's a bit obscure. What does it mean? If you're looking for the count of items, it would be more descriptive to have a method that looks like:

```ruby
expect(@audit_report_page).to have_content_item_count(1)
```